torch_xla/csrc/tensor.cpp

```diff
@@ -1489,7 +1514,9 @@ XLATensor XLATensor::sum(const XLATensor& input,
                          bool keep_reduced_dimensions,
                          c10::optional<at::ScalarType> dtype) {
   return input.CreateFrom(
-      ir::MakeNode<ir::ops::Sum>(input.GetIrValue(), std::move(dimensions),
+      ir::MakeNode<ir::ops::Sum>(input.GetIrValue(),
+                                 XlaHelpers::GetCanonicalDimensionIndices(
```

Comment:
Oh look! This can just swallow the std::vector directly w/out making any copies :)